File: commands/start.py


from telegram import InlineKeyboardButton, InlineKeyboardMarkup
import config


def start(bot, update):
    chat_id = update.message.chat.id
    keyboard = [[
        InlineKeyboardButton('Support Chat',
                             url=config.supportChatUrl)
    ],
        [
            InlineKeyboardButton('Android App Link',
                                 url=config.appUrl)
        ]]
    reply_markup = InlineKeyboardMarkup(keyboard)
    bot.sendMessage(chat_id, "<b>Hi, I Can Search Torrent Database For Your Query.</b>\n\n"
                             "Supports Inline Mode \n-/help For More Info\n",
                    parse_mode='HTML',
                    reply_markup=reply_markup)

File: commands/start.js
const config = require("../config");
const telebot = require("telebot");
const bot = new telebot(config.botToken);
const parseMode = config.parseMode;

const start = (msg) => {
  let replyMarkup = bot.inlineKeyboard([
    [
      bot.inlineButton("Support Chat", { url: "t.me/itorrentsupportchat" }),
      bot.inlineButton("Android App", {
        url:
          "https://play.google.com/store/apps/details?id=com.icodelife.itorrentsearch",
      }),
    ],
  ]);
  const startText = `<b>Hi, I Can Search Torrent Database For Your Query.</b>\nUse Command /find To Search\neg: /find ubuntu`;
  bot.sendMessage(msg.chat.id, startText, { parseMode, replyMarkup });
};
module.exports = start;
